code/wolfcare-main/src/backend/utils.py

```diff
@@ -0,0 +1,921 @@
+import mysql.connector
+from src.backend.dbconfig import constants
+import datetime
+
+try:
+    connection = mysql.connector.connect(
+        host=constants["host"], user=constants["user"], password=constants["password"], database=constants["database"])
+except:
+    pass
+
+def addDoctor(firstname, lastname, primaryspecialty, secondaryspecialty, type, degree, phone, email, gender, yoe, approvalstatus, isactive, userid):
+    """
+    Inserts doctor info into the database.
+
+    Parameters
+    ----------
+    firstname : string
+        First name of the doctor.
+    lastname : string
+        Last name of the doctor.
+    primaryspecialty : string
+        Primary specialty of the doctor. For eg. Dermatology, Neurology etc.
+    secondaryspecialty : string
+        Secondary specialty of the doctor, other than primaryspecialty. This can be NULL. For eg. Dermatology, Neurology etc.
+    type : string
+        Type of the doctor. For eg. Prescriber, Non Prescriber.
+    degree : int
+        Degree owned by the doctor. For eg. MBBS, MD etc.
+    phone : string
+        Contact number of the doctor. 
+    email : string
+        Email id of the doctor. 
+    gender : string
+        Gender of the doctor. For eg. Male, Female etc.
+    yoe : string
+        Years of experience of the doctor.
+    approvalstatus : string
+        Whether the doctor is approved by the admin. At time of registeration it will be False.
+    isactive : string
+        Whether the doctor is currently working or not. 
+    userid : int
+        ID of the doctor (Doctor also an user)
+
+
+    Returns
+    ----------
+    tuple
+        Returns a tuple with two elements. The first element(a boolean variable) checks to see if the database operations worked correctly. The second element is a message about the same.
+    """
+
+    try:
+        lastmoddate = str(datetime.datetime.today()).split()[0]
+        cursor = connection.cursor(dictionary=True)
+        mysql_insert_query = """INSERT INTO doctors (firstname, lastname, primaryspecialty, secondaryspecialty, type, degree, phone, email, gender, yoe, approvalstatus, isactive, lastmoddate, userid)
+                                VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s) """
+
+        record = (firstname, lastname, primaryspecialty, secondaryspecialty, type, degree,
+                  phone, email, gender, yoe, approvalstatus, isactive, lastmoddate, int(userid))
```

Comment:
Im guessing this approvalstatus should be set by us when we doctor is added.
Basically it should be set to FALSE.

Same with isActive =  it should be set to TRUE.

It wont be sent in the request.